spec/lib/email_alert_signup_api_spec.rb

```diff
@@ -66,6 +71,22 @@
     end
   end
 
+  context "when a default_frequency is provided" do
+    let(:default_frequency) { "daily" }
+
+    it "returns the url email-alert-api gives back, and appends the default_frequency param" do
+      subscription_url = "http://gov.uk/email/some-subscription"
+      init_simple_email_alert_api subscription_url
+      expect(subject.signup_url).to eql "http://gov.uk/email/some-subscription?default_frequency=daily"
+    end
+
+    it "appends the default_frequency param with an ampersand if other URL parameters exist" do
+      subscription_url = "http://gov.uk/email/some-subscription?foo=bar"
+      init_simple_email_alert_api subscription_url
+      expect(subject.signup_url).to eql "http://gov.uk/email/some-subscription?foo=bar&default_frequency=daily"
```

Comment:
There's an assumption here that `default_frequency` will always be the last parameter in the query string.  Whilst that's true at the moment, later refactors may mean that's not always the case.

Maybe `Rack::Utils.parse_query` would be better here, so the parameters can be turned into an array and parsed, e.g. https://stackoverflow.com/a/23923728.